pkg/fileutils/read_test.go

```diff
@@ -204,67 +204,76 @@ var _ = Describe("ReadUrl", func() {
 var _ = Describe("EmptyScheme", func() {
 
 	var (
+		file	string
 		u       *url.URL
+		os      string
 		err     error
 		empty   bool
 	)
 
+	BeforeEach(func() {
+		os = "linux"
```

Comment:
It would be good to put this in a "linux" Context and have a corresponding "windows" context. Then it will be easier to compare which tests are run for which OS's.